src/vl-search.js

```diff
@@ -164,6 +172,16 @@ export class VlSearch extends VlElement(HTMLElement) {
             </button>
         `);
     }
+
+    __processInputSlot() {
+        const slot = this.querySelector('[slot="input"]');
+        if (! slot) {
+            this._shadow.querySelector('slot[name="input"]').remove();
```

Comment:
Kan je de default `<input>` niet gewoon onder `<slot>` steken en de content door de browser laten vervangen als men een slot aanlevert? Dan moet je het verwijderen niet doen, lijkt me.

Reply:
+1

Reply:
Als we dat deden, kregen we de stijl niet meer goed.

Reply:
Werd de css voor class `vl-search__input` niet toegepast dan, bedoel je?

Reply:
Geen idee meer. Ik heb het daarnet nog eens geprobeerd zoals je voorstelde, maar kreeg het niet in orde. Gans in het begin ben ik ook zo begonnen, uiteindelijk zijn we tot de oplossing gekomen zoals ze nu is.
Als je de input in de slot zet, klopt de stijl van het 'origineel' niet meer. Het veld waarin je tekst moet intypen neemt dan niet meer dezelfde breedte in beslag.

Reply:
Het voelt minder elegant aan op deze manier, but if it works, I guess... :)